api/services/wwwConfig.js

```diff
@@ -123,12 +123,10 @@ function _getAPISystemIntegration(bag, next) {
             'No systemIntegration found for query: ' + query)
         );
 
-      var defaultAPIIntegration = _.findWhere(
+      bag.publicApiIntegration = _.findWhere(
         systemIntegrations, {name: 'api'});
-      var serviceAPIIntegration = _.findWhere(
```

Comment:
Do we want to remove this? I'm not quite sure what the original purpose of this was.

Reply:
I think this was there to make sure www uses the api Url from the systemIntegration defined in config.json